tests/test_suite1/conftest.py

```diff
@@ -0,0 +1,2 @@
+from tests.fixtures.drivers import *
+from tests.fixtures.pages import *
```

Comment:
will it work if you move that import lines to __init__.py ?
so you could become more simple.
...i'm not sure it will work, so please check it by your own :)

Reply:
No...It's a special file. It should be named exactly conftest.py. If use import in the conftest.py, all became imported in all files located in the same dir as  conftest.py